wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -682,11 +682,6 @@ public void onCreate() {
 
         super.onCreate();
 
-        application = (WalletApplication) getApplication();
```

Comment:
this like is very critical, otherwise `application = null`

Reply:
Good catch!